action.yml

```diff
@@ -18,3 +18,6 @@ inputs:
     required: false
   output_path:
     description: "Output path for the decorated OpenAPI spec."
+  project_name: 
+    description: "Stainless project name."
+    required: true
```

Comment:
I don't think this should be required as our existing customers don't have to provide it right?

Reply:
I do think that we want existing customers to migrate and add this, so we may need to send out comms to anyone using it. But we can also discuss a specific action plan for this as a team before merging this update.

Reply:
Why do we need this?

Reply:
(reason I ask is that breaking changes to an API is a bad look for an API company, more work for customers is not great in general, and if a customer only has one project I would assume we'd be able to infer it?)

Reply:
that's a good point, I can update to make it optional

Reply:
> reason I ask is that breaking changes to an API is a bad look for an API company, more work for customers is not great in general

On the other hand I would argue it is preferable to do that change early on, when the number of customers to reach out to is still manageable.

I think it's safe to make it optional, but document it as the default way to use the action. And mention in the parameter description that it may become required in the future.

That gives us the option to contact customers that would need to update their workflow, then enforce the use of the project name, if we see it as necessary.